docs/usage/channels.rst

```diff
@@ -11,115 +11,143 @@ Channels provide:
 
 1. Independent :term:`broker` backends, optionally handling inter-process communication
    and data persistence on demand
-2. "Channel" based subscription management
+2. "Channel" based :term:`subscription` management
```

Comment:
Seems the glossary section needs to be expanded for the term links to work. Can that be expanded by default?

Reply:
It can but if it I think i'd prefer it at the bottom of the page since its chunky